doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,321 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
 
 <a name="9.4"></a>
 ## 9.4 Adoption Roadmap
+
+<a name="9.5"></a>
+## 9.5 CNTT Field Trial/ Approach
+
+This portion of Chapter 9 is segmented into two subsections. Section 9.5.1 provides a summary and overview of the trials activities specifically targeted to potential trials participants. Section 9.5.2 addresses the overall approach of CNTT to field trials as a method of ensuring consistency between releases.
+
+<a name="9.5.1"></a>
+
+### 9.5.1 Summary/Field Trials Participants Overview
+
+CNTT has built a Reference Architecture (RA-1) on which Reference Implementation (RI-1) and Reference Conformance (RC-1) requirements have been defined. To ensure value add to Telco industry operators, suppliers, and end user customers, it is running field tests to validate the feasibility, utility, and effectiveness of its requirements and methods (RI-1/RC-1).
+
+<a name="9.5.1.1"></a>
+#### 9.5.1.1 Intents of Field Trials
+
+This is a partnership of CNTT with participants to validate the community is adopting a consistent approach. This is not a VI badging exercise. The trials will validate the RI-1 and the RC-1 test suite requirements and methods themselves, not VNFs or VI systems under test. 
```

Comment:
They would rather validate RA1 ad RC1. RI1 should pass RC1 as the OpenStack deployments part of the trials.

Reply:
If the last sentence is replaced with: "The trials will validate that the RC-1 test suite is sufficient to validate that RI-1 conforms to RA-1, and evaluate the process for testing and results submission. VNFs and VIs are not under test." - will that align better?

Reply:
From my understanding RI1 is already continuously verified by RC1 in OPNFV out of the field trial. By running locally RC, we check conformance of third-party NFVI deployments with RA1. The VNF certification part would leverage on RI1 but I don't think it's part of the field trials.

Reply:
@ss8171 I prefer your rewrite.